### PR TITLE
Ana/add treasury proposals thresholds

### LINKED ISSUE #4988: Show thresholds for proposals
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

Currently proposals only show votes or deposits. We don't show required thresholds to succeed or pass.

I.e. in PolkadotJs
image

**Describe alternatives you've considered**

Display a bar that shows current votings/deposits and the thresholds. Negative votes could come from the right so we can show the threshold for those on the right. @jbibla ideas?

**Additional context**

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4988

**Description:**

All good

image

I can make the threshold span more prominent though

image

EDIT: Threshold is not displayed anymore

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f6c98f23f1f771a0115ba3c--lunieio.netlify.app

**Comment by Bitcoinera** [comment]
WIP since there is a decision to be made on the display for Nays

File: app/src/components/governance/ProposalStatusBar.vue [hunk 1]
```diff
@@ -26,13 +26,23 @@
...
+          <span>Threshold:</span>
+          <span>{{ proposal.detailedVotes.votingThresholdYes }} Votes</span>
+        </div>
       </div>
       <ProgressBar
         size="large"
-        :val="votePercentage * 100"
+        :val="
+          proposal.type === 'TREASURY'
+            ? proposal.detailedVotes.votingThresholdYes
```
Comment (faboweb) [inline app/src/components/governance/ProposalStatusBar.vue:43]: this shows the threshold not the current status am I wrong? Shouldn't it be: yesVotes / votingThresholdYes?

Comment (Bitcoinera) [reply app/src/components/governance/ProposalStatusBar.vue:43]: We could actually display both. But the latter is already visually displayed on the progress bar. But we could this display it as a percentage too :shrug: Not a bad idea

Comment (faboweb) [reply app/src/components/governance/ProposalStatusBar.vue:43]: let's have a call on this

Comment (Bitcoinera) [reply app/src/components/governance/ProposalStatusBar.vue:43]: Tuesday?

**Comment by jbibla** [comment]
this should be implemented as a bar for all proposals. but also, this is not a priority right now.

**Comment by Bitcoinera** [comment]
> this should be implemented as a bar for all proposals. but also, this is not a priority right now.

The others don't have voting threshold, or you mean Yes/No bars (two bars instead of one)?

It might not be a priority right now but it would nice to wrap governance up, with the cherry on the top, don't you think?

**Comment by jbibla** [comment]
everything has a threshold (some measure of when the state will change). for treasury proposals it's a majority of council members for example.

**Comment by Bitcoinera** [comment]
> everything has a threshold (some measure of when the state will change). for treasury proposals it's a majority of council members for example.

In Polkadot referendums and treasuries, yes. But in the case of Cosmos I thought there is merely a deadline, and when this is reached it is decided upon the votes percentage at that point

**Comment by jbibla** [comment]
> in the case of Cosmos I thought there is merely a deadline, and when this is reached it is decided upon the votes percentage at that point

yes, true. but at the deadline the state change is determined by the voting outcomes which we can display as a threshold.

**Comment by Bitcoinera** [comment]
> yes, true. but at the deadline the state change is determined by the voting outcomes which we can display as a threshold.

The threshold is always simple majority in Cosmos, right?

**Comment by jbibla** [comment]
> The threshold is always simple majority in Cosmos, right?

i don't know - please confirm with the cosmos docs!

**Comment by Bitcoinera** [comment]
'kay, will have a look

**Comment by Bitcoinera** [comment]
> please confirm with the cosmos docs!

It seems it is this way: https://docs.cosmos.network/v0.40/modules/gov/01_concepts.html#threshold

The only two things to take into account is that the proposal won't be valid if more than 1/3rd votes NoWithVeto and also a minimum Quorum must be reached. But we don't need to display that for now. Threshold is this what stands on the docs

File: app/src/components/governance/ProposalStatusBar.vue (hunk 1 above)
Comment (Bitcoinera) [reply app/src/components/governance/ProposalStatusBar.vue:43]: oops, we forgot about this, didn't we?

**Comment by Bitcoinera** [comment]
Seems there are still some issues with Polkadot proposals times :eyes: 

image

**Comment by Bitcoinera** [comment]
:thinking: 

image

 Guess we are missing something here

File: app/src/components/governance/ProposalStatusBar.vue [hunk 2]
```diff
@@ -31,18 +31,14 @@
           <span>{{ votePercentage | percentInt }} of {{ stakingDenom }}</span>
           <span v-if="voteCount">({{ voteCount }} Votes)</span>
         </div>
-        <div v-if="true" class="vote-data">
+        <div v-if="proposal.type === governanceStatusEnum.TREASURY" class="vote-data">
```
Comment (jbibla) [inline app/src/components/governance/ProposalStatusBar.vue:34]: i don't understand why thresholds are only for treasury proposals...? every proposal type has a threshold.

**Comment by Bitcoinera** [comment]
Alright, so I removed the simple display span for thresholds I added and this PR what is currently doing it simply fixing voting percentages in detailedVotes and in PropsalStatusBar

**Comment by Bitcoinera** [comment]
Rerunning E2E

**Review by jbibla — APPROVED** [review]

**Comment by jbibla** [comment]
thanks ana!

**Pull request merged**